lib/generators/hyrax/templates/db/migrate/20240606205216_add_contexts_to_hyrax_flexible_schemas.rb.erb

```diff
@@ -0,0 +1,5 @@
+class AddContextsToHyraxFlexibleSchemas < ActiveRecord::Migration<%= migration_version %>
+  def change
```

Comment:
Should this migration have a conditional check if it already exists?

Reply:
yep